traffic_ops/client/deliveryservice.go

```diff
@@ -134,7 +134,7 @@ func (to *Session) GetDeliveryServices() ([]tc.DeliveryService, ReqInf, error) {
 func (to *Session) GetDeliveryServicesByServer(id int) ([]tc.DeliveryService, ReqInf, error) {
 	var data tc.DeliveryServicesResponse
 
-	reqInf, err := get(to, fmt.Sprintf(API_SERVER_DELIVERY_SERVICES, id), &data)
+	reqInf, err := get(to, API_DELIVERY_SERVICES + "?id=" + strconv.Itoa(id), &data)
```

Comment:
I think this one needs to be reverted.  This should use /servers/{id}/deliveryservices so shouldnt need to be changed for this PR